totaladapters/src/test/java/pl/arturborowy/adapters/gridview/base/BaseGridViewAdapterTest.kt

```diff
@@ -27,8 +26,23 @@ internal class BaseGridViewAdapterTest : AdapterTest() {
     }
 
     @Test
-    fun `getCount returns items size`() {
-        Assert.assertEquals(givenStrings.size, adapter.count)
+    fun `lambda notifyDataSetChanged invoke calls notifyDataSetChanged() from BaseAdapter`() {
```

Comment:
I think `invoke` is redundant here.

Reply:
It's not important to me

Reply:
What? I don't understand. I'm just saying this sentence doesn't make sense in english and it should probably be `lambda notifyDataSetChanged calls notifyDataSetChanged() from BaseAdapter` or `notifyDataSetChanged lambda invocation calls notifyDataSetChanged() from BaseAdapter`.

Reply:
I meant "invoke" as method (my bad), but now i think "invocation" matches better.